components/org.wso2.carbon.identity.tag.manager/src/main/java/org/wso2/carbon/identity/tag/manager/exception/TagServiceClientException.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.com).
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.tag.manager.exception;
+
+import org.apache.commons.lang.StringUtils;
+import org.wso2.carbon.identity.tag.manager.model.ErrorMessage;
+
+/**
+ * This class contains the implementation for the client exceptions.
+ */
+public class TagServiceClientException extends TagServiceException {
+
+    /**
+     * Constructs a new exception with the specified detail message.
+     *
+     * @param message The detail message
+     */
+    public TagServiceClientException(String message) {
+
+        super(message);
+        this.setErrorCode(getDefaultErrorCode());
+    }
+
+    /**
+     * Constructs a new exception with the specified detail message and cause.
+     *
+     * @param message The detail message
+     * @param cause   The cause
+     */
+    public TagServiceClientException(String message, Throwable cause) {
+
+        super(message, cause);
+        this.setErrorCode(getDefaultErrorCode());
+    }
```

Comment:
Lets remove all the constructors with no error codes. We should encourage all to use error codes

Reply:
done